zboss/production/include/osif/zb_config_platform.h

```diff
@@ -1,7 +1,7 @@
 /*
  * Copyright (c) 2020 Nordic Semiconductor ASA
  *
- * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
```

Comment:
Please fix the license tag (revert) and update date.

Reply:
License and copyright year fixed